client/src/Dataset/List/list.hooks.ts

```diff
@@ -82,17 +180,27 @@ const useList = (itemsPerPage: number): ListState => {
             setDatasets(data.datasets);
             setCategories(data.categories);
             setPageCount(data.pagination.pages);
-
+            if (currentUser && currentUser.is_admin) {
+                const {
+                    data: { users },
+                } = await AdminPanelApi.getUsers(itemsPerPage);
+                setUsers(users);
+            }
             removeProcess(dispatch, process);
         };
         update(page);
-    }, [page, generation, itemsPerPage, dispatch]);
+    }, [page, generation, itemsPerPage, dispatch, currentUser]);
+
+    const tags = categories.map(category => category.name);
+    const usernames = users.map(user => user.username);
 
     return {
-        page: page,
+        offset: offset,
```

Comment:
use es6 short keys in objects (no need to repeat key and value if the name is the same)